internals/secrethub/audit.go

```diff
@@ -63,13 +90,34 @@ func (cmd *AuditCommand) run() error {
 		return err
 	}
 
-	tabWriter := tabwriter.NewWriter(cmd.io.Stdout(), 0, 4, 4, ' ', 0)
-	header := strings.Join(auditTable.header(), "\t") + "\n"
-	fmt.Fprint(tabWriter, header)
+	var formatter tableFormatter
+	if cmd.json {
+		formatter = newJSONFormatter(auditTable.header())
+	} else {
+		terminalWidth, err := cmd.terminalWidth(int(os.Stdout.Fd()))
+		if err != nil {
+			terminalWidth = defaultTerminalWidth
+		}
+		formatter = newColumnFormatter(terminalWidth, auditTable.columns())
+	}
+
+	paginatedWriter, err := cmd.newPaginatedWriter(os.Stdout)
+	if err == errPagerNotFound {
+		paginatedWriter = newFallbackPager(os.Stdout)
+	} else if err != nil {
+		return err
+	}
+	defer paginatedWriter.Close()
+
+	if formatter.printHeader() {
```

Comment:
The function name `printHeader` suggests that it does something, but it actually only checks if it should print the header, right? So consider naming it `shouldPrintHeader` instead.